tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -8,7 +8,11 @@ esac
 done
 
 DEBIAN_FRONTEND=noninteractive
-apt-get update && apt-get install -y software-properties-common
+mkdir -p /tmp/dotnet
+aria2c -q -d /tmp/dotnet https://packages.microsoft.com/config/ubuntu/16.04/packages-microsoft-prod.deb
```

Comment:
Is it possible to not hard code 16.04 at here? So that we can reuse this script for ubuntu 18.04?

Reply:
Just a suggestion. You don't have to solve it.